symphony/bdk/core/client/api_client_factory.py

```diff
@@ -71,11 +74,34 @@ def _get_api_client(self, server_config, context) -> ApiClient:
         if server_config.proxy is not None:
             self._configure_proxy(server_config.proxy, configuration)
 
-        return ApiClient(configuration=configuration)
+        client = ApiClient(configuration=configuration)
+        client.user_agent = self._user_agent()
+
+        return client
 
     @staticmethod
     def _configure_proxy(proxy_config, configuration):
         configuration.proxy = proxy_config.get_url()
 
         if proxy_config.are_credentials_defined():
-            configuration.proxy_headers = urllib3.util.make_headers(proxy_basic_auth=proxy_config.get_credentials())
+            configuration.proxy_headers = urllib3.util.make_headers(proxy_basic_auth=proxy_config.get_credentials(),
+                                                                    user_agent=ApiClientFactory._user_agent())
+        else:
+            configuration.proxy_headers = urllib3.util.make_headers(user_agent=ApiClientFactory._user_agent())
+
+    @staticmethod
+    def _user_agent():
+        return f"Symphony-BDK-Python/{ApiClientFactory._bdk_version()} Python/{ApiClientFactory._python_version()}"
```

Comment:
The format looks a bit different from the Java one:

return "Symphony BDK/" + getBdkVersion() + "/java/" + System.getProperty("java.version");

Reply:
Yes but I realized we should not have spaces : https://tools.ietf.org/html/rfc7231#section-5.5.3
I was thinking this was more in line with the RFD. WDYT ?

Reply:
I agree, let's update it for the Java BDK then (and consider it a non breaking change...)

Reply:
GH issue opened here on the Java BDK: https://github.com/finos/symphony-bdk-java/issues/471